edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -108,7 +109,8 @@ def authenticate(self, request):
             # .. custom_attribute_name: jwt_auth_failed
             # .. custom_attribute_description: Includes a summary of the JWT failure exception
             #       for debugging.
-            set_custom_attribute('jwt_auth_failed', 'Exception:{}'.format(repr(exception)))
+            exception_to_report = _lowest_jwt_exception(exception)
```

Comment:
Nit: Consider 'deepest' over 'lowest', but this is really just a nit.

Reply:
Good idea, I'll update.